taichi/backends/vulkan/aot_module_builder_impl.h

```diff
@@ -21,6 +21,11 @@ class AotModuleBuilderImpl : public AotModuleBuilder {
   void dump(const std::string &output_dir,
             const std::string &filename) const override;
 
+  // FIXME: remove me once TaichiAotData is no longer backend specific.
```

Comment:
I think it's fine to add `ti_aot_data_` to the module builder base class. It's just that not every impl uses it, yet?